### DISCUSSION TITLE
How to swich off info messages?

### DISCUSSION THREAD

**Discussion by Pedklin** [etna-team/etna#510] [Q&A] [answered]
Is it possiple to supress the info messages of models like this:
![rnn_log](https://github.com/user-attachments/assets/3d0ba6ae-7494-475f-a759-b3f4f9b88f6a)
They are saved in the log of our system and users get scared

**Comment by d-a-bunin** [comment, marked as answer]
Hi! Is it a backtest that produces such logs? 

It looks like these logs are produced by `joblib`. You could pass `joblib_params=dict(verbose=0, backend="multiprocessing", mmap_mode="c")` into `backtest` to silence them.

**Reply by Pedklin** [reply]
Ok, thank you, it works
